examples/docs_snippets/docs_snippets/concepts/resources/pythonic_resources.py

```diff
@@ -57,15 +63,14 @@ def new_resources_assets_defs() -> "dg.Definitions":
     def data_from_url(data_url: dg.ResourceParam[str]) -> dict[str, Any]:
         return requests.get(data_url).json()
 
-    defs = dg.Definitions(
-        assets=[data_from_url],
-        resources={"data_url": "https://dagster.io"},
-    )
+    @dg.definitions
+    def resources():
+        return dg.Definitions(
+            resources={"data_url": "https://dagster.io"},
+        )
 
     # end_new_resources_assets_defs
 
-    return defs
```

Comment:
@yuhan it would be great to get some help on the best way to update the docs test for the resources. Many of the code snippets are changed to remove the `defs` returned